src/theme.ts

```diff
@@ -7,6 +7,7 @@ const colors = {
   secondary: '#534E71',
   secondaryLight: '#7378A3',
   text: '#534E71',
+  lightText: '#FFFFFF',
```

Comment:
```suggestion
  textLight: '#FFFFFF',
```

For consistency with other colour names.